satpy/etc/composites/vii.yaml

```diff
@@ -107,3 +107,40 @@ composites:
           - name: 'vii_8540'
       - name: 'vii_10690'
     standard_name: dust
+
+  true_color_daytime:
+      description: >
+        True Color during daytime
+      compositor: !!python/name:satpy.composites.fill.DayNightCompositor
+      standard_name: image_ready
+      lim_low: 78
+      lim_high: 88
+      day_night: day_only
+      prerequisites:
+        - true_color
+
+  true_color_daytime_uncorrected:
+      description: >
+        True Color during daytime
+      compositor: !!python/name:satpy.composites.fill.DayNightCompositor
+      standard_name: image_ready
+      lim_low: 78
+      lim_high: 88
+      day_night: day_only
+      prerequisites:
+        - true_color_uncorrected
+  
+  night_10695:
```

Comment:
do you need this one? I think you can use the channel `vii_10690` directly, if you don't need a specific enhancement for it